source/FreeRTOS_Routing.c

```diff
@@ -241,10 +241,6 @@ struct xIPv6_Couple
     {
         NetworkEndPoint_t * pxIterator = NULL;
 
-        /* This end point will go to the end of the list, so there is no pxNext
-         * yet. */
-        pxEndPoint->pxNext = NULL;
-
```

Comment:
But i think we should also add pxEndPoint->pxNext = NULL; in line 262 in case there is no endpoint in the list

Reply:
Also can we move [commit](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1020/commits/91badc97b5a4073f7e43d65e35fa094acddff16c) as a separate PR?